deepchecks/vision/suites/default_suites.py

```diff
@@ -61,8 +60,6 @@ def train_test_validation(n_top_show: int = 5,
 
     Parameters
     ----------
-    n_top_show : int, default: 5
```

Comment:
why remove that?

Reply:
Ambiguity at the suite level, also was not available for all suites